.github/workflows/Ci.yml

```diff
@@ -0,0 +1,102 @@
+name: "CI: Run tests"
+
+# GitHub Actions triggers based on the PR's and each commits on develop and master
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [develop, master]
+
+jobs:
+  build:
+    defaults:
+      run:
+        shell: bash
+        working-directory: "/opt/carma/"
+
+    # Specify the runner environment
+    runs-on: ubuntu-latest
+
+    # Set up a Docker container for the job
+    container:
+      image: usdotfhwastoldev/autoware.ai:develop
+      env:
+        INIT_ENV: "/home/carma/.base-image/init-env.sh"
+        ROS_2_ENV: "/opt/ros/foxy/setup.bash"
+        TERM: xterm
+      options: "--user root"
+
+    steps:
+    - name: Checkout ${{ github.event.repository.name }}
+      # Check out the repository code
+      uses: actions/checkout@v3.3.0
+      with:
+        path: src/${{ github.event.repository.name }}
+        fetch-depth: 0
+
+    - name: Move source code
+      # Move the source code to the desired location
+      run: mv $GITHUB_WORKSPACE/src /opt/carma/
+
+    - name: Checkout dependencies
+      # Checkout project dependencies
+      run: |
+        source "$INIT_ENV"
+        ./src/${{ github.event.repository.name }}/docker/checkout.bash -r /opt/carma/
+
+    - name: Build ROS1
+      # Build ROS1 packages

```

Comment:
We may eventually need a better method of listing all ROS 1 packages that should be skipped so that we don't need to list them in different locations. But for now, could you add a comment here (and also under 'Run ROS 1 C++ Tests') stating "The skipped ROS 1 packages here should match the ROS 1 packages that are skipped during the build in docker/install.sh"

Reply:
Added a comment